packages/expo-battery/src/ExpoBattery.web.ts

```diff
@@ -1 +1,97 @@
-export default {};
+import { EventEmitter } from '@unimodules/core';
+import { canUseDOM } from 'fbjs/lib/ExecutionEnvironment';
+
+import { BatteryState } from './Battery.types';
+
+const emitter = new EventEmitter({} as any);
+
+declare var navigator: Navigator;
+
+interface BatteryManager extends BatteryManagerEventTarget {
+  readonly charging: boolean;
+  readonly chargingTime: number;
+  readonly dischargingTime: number;
+  readonly level: number;
+}
+
+interface BatteryManagerEventTargetEventMap {
+  chargingchange: Event;
+  chargingtimechange: Event;
+  dischargingtimechange: Event;
+  levelchange: Event;
+}
+
+interface BatteryManagerEventTarget extends EventTarget {
+  onchargingchange: (this: BatteryManager, ev: Event) => any;
+  onlevelchange: (this: BatteryManager, ev: Event) => any;
+  onchargingtimechange: (this: BatteryManager, ev: Event) => any;
+  ondischargingtimechange: (this: BatteryManager, ev: Event) => any;
+  addEventListener<K extends keyof BatteryManagerEventTargetEventMap>(
+    type: K,
+    listener: (this: BatteryManager, ev: BatteryManagerEventTargetEventMap[K]) => any,
+    useCapture?: boolean
+  ): void;
+}
+
+async function getBatteryManagerAsync(): Promise<BatteryManager | null> {
```

Comment:
Move this and the other helper functions below the `export default {...` bit so that when someone reads this file, they see the more important things like the public API and data structures first, and the implementation details second.